docs/gno-tooling/cli/gnoland.md

```diff
@@ -4,28 +4,88 @@ id: gno-tooling-gnoland
 
 # gnoland
 
-## Run a Gnoland Node
+## Overview
 
-Start a node on the Gnoland blockchain with the following command.
+`gnoland` is the Gno.land blockchain client binary, which is capable of managing node working files, as well
+as starting the blockchain client itself.
 
-```bash
-gnoland
+```shell
+gnoland --help
+
+USAGE
+  <subcommand> [flags] [<arg>...]
+
+starts the gnoland blockchain node.
+
+SUBCOMMANDS
+  start    run the full node
+  secrets  gno secrets manipulation suite
+  config   gno config manipulation suite
+  genesis  gno genesis manipulation suite
+  init     initializes the default node secrets / configuration
```

Comment:
also move here above start

Reply:
This wouldn't be a problem with an automated tool 🤦‍♂️ 

I've dropped it entirely, until we figure out if / how to automate this in the docs:

[a2da48d](https://github.com/gnolang/gno/pull/1985/commits/a2da48d7a023dbc3b2f5b8e6dfbbffe3d4c322ce)